.docker/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# mount your private key for bitbucket into the container to allow this command to succeed
+pip install --trusted-host pypi.python.org -r requirements.txt
+
+export PINERY_URL='http://seqbio-pinery-prod-www.hpc.oicr.on.ca:8080/pinery-ws-miso/'
+export GSI_ETL_CACHE_DIR='/cache'
+export MONGO_URL="mongodb://provenance_ro:$(cat /run/secrets/mongo_password)@provenance-mongo-db.gsi.oicr.on.ca:27017/provenance"
+export GSI_QC_ETL_RUNSCANNER_MASTER_URL='http://runscanner.res.oicr.on.ca:8080/'
```

Comment:
It turns out the run scanner URL should be `http://runscanner.res.oicr.on.ca:8080/runs/all`. (You could omit it here if you wanted since it's only used for building the cache; Dash should only use prebuilt caches) (Savo is updating the gsi-qc-etl documentation to reflect this)

Reply:
Removed.